### PR TITLE
Please add this policy setting.

### PR DESCRIPTION
**Pull request by yogeshasalkar**
Hi, Please add this policy setting to Local Policies security options. It was missing in the documentation. But can be configured via Intune.
#6651 

Thanks

### CODE REVIEW

**Comment by yogeshasalkar** [comment]
Hi @JohanFreelancer9 could you please have a look?

File: windows/client-management/mdm/policy-csp-localpoliciessecurityoptions.md [hunk 1]
```diff
@@ -2385,6 +2388,74 @@ GP Info:
...
+This security setting allows a client device to require the negotiation of 128-bit encryption and/or NTLMv2 session security. These values are dependent on the LAN Manager Authentication Level security setting value. The options are:
+
+Require NTLMv2 session security: The connection will fail if message integrity is not negotiated.
+Require 128-bit encryption. The connection will fail if strong encryption (128-bit) is not negotiated.
+
+Default:
+
+Windows XP, Windows Vista, Windows 2000 Server, Windows Server 2003, and Windows Server 2008: No requirements.
+
+Windows 7 and Windows Server 2008 R2: Require 128-bit encryption
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/policy-csp-localpoliciessecurityoptions.md:2446]:
```suggestion
Windows 7 and Windows Server 2008 R2: Require 128-bit encryption.
```

File: windows/client-management/mdm/policy-csp-localpoliciessecurityoptions.md [hunk 2]
```diff
@@ -2385,6 +2388,74 @@ GP Info:
...
+<hr/>
+
+<!--/Scope-->
+<!--Description-->
+Network security: Minimum session security for NTLM SSP based (including secure RPC) clients
+
+This security setting allows a client device to require the negotiation of 128-bit encryption and/or NTLMv2 session security. These values are dependent on the LAN Manager Authentication Level security setting value. The options are:
+
+Require NTLMv2 session security: The connection will fail if message integrity is not negotiated.
+Require 128-bit encryption. The connection will fail if strong encryption (128-bit) is not negotiated.
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/policy-csp-localpoliciessecurityoptions.md:2440]:
```suggestion
Require 128-bit encryption: The connection will fail if strong encryption (128-bit) is not negotiated.
```

**Comment by ManikaDhiman** [comment]
Merging this PR now. I will create a new PR to add this policy to the [Policy CSP](https://docs.microsoft.com/en-us/windows/client-management/mdm/policy-configuration-service-provider) index page and What's new section for [Windows 10, version 1803.](https://docs.microsoft.com/en-us/windows/client-management/mdm/new-in-windows-mdm-enrollment-management#whats-new-in-mdm-for-windows-10-version-1803). Thanks!

**Comment by e0i** [comment]
Closing via merged PR #6651. Thank you.

**Pull request merged**